applications/FluidDynamicsApplication/python_scripts/fluid_dynamics_analysis.py

```diff
@@ -8,6 +8,9 @@
 from KratosMultiphysics.process_factory import KratosProcessFactory
 from KratosMultiphysics.FluidDynamicsApplication import python_solvers_wrapper_fluid
 
+# Import auxiliar methods
+from KratosMultiphysics.auxiliar_methods_adaptative_remeshing import AuxiliarMethodsAdaptiveRemeshing
```

Comment:
thie should be approved by the flui dynamics team

@rubenzorrilla @jcotela you ok with this?

Reply:
Before any specific approval I'd rather discuss this in the @KratosMultiphysics/technical-committee .